randovania/games/samus_returns/logic_database/Area 5 - Exterior.txt

```diff
@@ -240,6 +247,15 @@ Extra - asset_id: collision_camera_002
                           Phase Drift and Damage Boost (Beginner) and Movement (Intermediate)
                       Phase Drift or Spider Ball or Phase Drift Skip (Expert)
 
+> Right Side Middle Ledge; Heals? False
+  * Layers: default
+  > Door to Zeta Arena Access
+      High Jump Boots or Space Jump or Single-wall Wall Jump (Intermediate) or Use Spider Ball
```

Comment:
Can't you just jump from the ledge below? If not, you could probably IBJ to the Door? 
![image](https://github.com/randovania/randovania/assets/117127188/aac3cfe5-4435-4992-9d7c-b27426368ade)

Reply:
With High Jump definitely. Bombs would also work

Reply:
Just tested, it's too far to make with a single jump

Reply:
If Bombs work they shoud be added or am I missing something? Diagonal Bomb Jump but as you could just fall down to the ammo recharge station a simple ibj should be fine.

Reply:
Yeah the bomb requirement is handled by the IBJ at the bottom. Could add a diagonal req at the top but thats kinda unecessary.

Reply:
Hmmm I serioulsy cannot find a logical way with IBJ to reach that door. There is only a connection from `Right Side Middle Ledge` to it?

Reply:
![image](https://github.com/randovania/randovania/assets/38679103/8041ca31-a82b-4f43-b150-9b66fad0e06f)

Reply:
I'm confused. Is your state of the screenshot really in this branch?
![image](https://github.com/randovania/randovania/assets/117127188/d893e077-1184-4f73-85ed-dc03fa9b8832)
![image](https://github.com/randovania/randovania/assets/117127188/7ac916e0-49a3-4c6e-86c6-f1dc383931be)

Reply:
Somehow it got pushed to the wrong branch. Ill push this real quick.